src/docs/sql-editor/nft-tables/index.md

```diff
@@ -0,0 +1,313 @@
+
+Below we have listed all the nft tables with thier columns, a brief descriptions and some example queries which you can use to explore the tables.
+
+### nft trades
+
+The `nft_trades` is a table designed to store information about the NFT trades that we have collected. It has the following columns:
+
+- **dt** (*DateTime*): When the transaction happened.
+
+- **blockchain** (*String*): On which blockchain.
+
+- **tx_hash** (*String*): Unique identifier that is generated when the transaction was executed.
+
+- **log_index** (*UInt32*): Log index of trade event. 
+
+- **platform** (*String*): Name of platform where trades occured such as opensea, cryptopunks and etc. 
+
+- **seller_address** (*String*): Address who sold the nft.
+
+- **buyer_address** (*String*): Address who bought the nft.
+
+- **nft_contract_address** (*String*): Address of the nft's collection.
+
+- **currency_contract_address** (*Nullable(String)*): Address of ERC20 token or empty if native ether is used.
+
+- **asset_ref_id** (*UInt64*): Reference ID of the currency contract that can be used to retrieve additional information about the asset from the `asset_metadata` table.
+
+- **token_ids** (*Array(String)*): Array ids of NFT token in hex.
+
+- **amount_tokens** (*Array(String)*): Array of amounts, which describes how many tokens were transfered.
+
+- **nft_contract_type** (*UInt8*): Type of nft contract, 0 - erc721, 1 - erc1155. Based on signature of transfer.
+
+- **amount** (*String*): The amount of the transaction in the asset used in it.
+
+- **complete** (*UInt8*): State of the transaction. (1 means it is completed).
+
+- **computed_at** (*DateTime*): The timestamp when the trade was inserted to the table.
+
+### nft tokens metadata
+
+The `nft_tokens_metadata` is a table designed to store information about NFT tokens. It has the following columns:
+
+- **blockchain** (*String*): Refers to the decentralized platform on which the NFT exists, e.g. Ethereum, Binance Smart Chain, etc.
+
+- **address** (*String*): Represents the unique public address of the NFT owner.
+
+- **token_id** (*String*): Unique identifier assigned to each NFT, used to distinguish it from others.
+
+- **uri** (*Nullable(String)*): Uniform Resource Identifier, a link that directs to additional information or media related to the NFT.
+
+- **data** (*Nullable(String)*): Contains the NFT's metadata, such as its name, image, and any other relevant information.
+
+- **created_at_** (*DateTime*): The date and time when the NFT was created.
+
+- **error** (*Nullable(String)*): Any error message associated with the NFT, such as a transaction failure, if applicable. This column is used to track and monitor any issues that arise in the NFT creation process.
+
+### intraday nft metrics
+
+The `intraday_nft_metrics` is a table designed to store intraday metrics for NFT assets. The table has the following columns:
+
+- **asset_id** (*UInt64*): Unique identifier assigned to the NFT asset.
```

Comment:
Looking at the NFT metrics article I suspect that this is not correct. asset_id is probably the id of Ether or WETH. Please doublecheck